### PR TITLE
revise label for CL_0000047 (neural stem cell)

### LINKED ISSUE #714: neuronal stem cell logical definition is wrong
text: Neural stem cell is characterized as an undifferentiated cell that originates from the neuroectoderm and has the capacity both to perpetually self-renew without differentiating and to generate multiple types of lineage-restricted progenitors.

'multi fate stem cell'
 and ('part of' some 'central nervous system')

BAD SMELL: text def massively differing from logical def. See S11 in https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/

The logical def is clearly wrong. Consider glial cells, or even resident immune cells, or blood vessel cells.

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
- revise definition: An undifferentiated neural cell that originates from the neuroectoderm and has the capacity both to perpetually self-renew without differentiating and to generate multiple central nervous system neuronal and glial cell types. (source: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6710163/)
- remove logical def
- remove asserted superclass 'multi fate stem cell'
- add superclass 'neural cell'

close #714

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -3524,17 +3524,16 @@ AnnotationAssertion(rdfs:comment obo:CL_0000046 "Consider using term 'pro-B cell
...
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30639325"^^xsd:string) obo:IAO_0000115 obo:CL_0000047 "An undifferentiated neural cell that originates from the neuroectoderm and has the capacity both to perpetually self-renew without differentiating and to generate multiple central nervous system neuronal and glial cell types.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000047 "BTO:0002881"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000047 "CALOHA:TS-2360"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000047 "FMA:86684"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000047 "NSC"^^xsd:string)
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000047 "neural stem cell"^^xsd:string)
-AnnotationAssertion(rdfs:label obo:CL_0000047 "neuronal stem cell"^^xsd:string)
-EquivalentClasses(obo:CL_0000047 ObjectIntersectionOf(obo:CL_0000048 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001017)))
-SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000047 obo:CL_0000048)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000047 "neuronal stem cell"^^xsd:string)
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:3534]: Was uncertain if this term also included the "glial cell types" mentioned in the definition, but appears that it used that way as well: https://doi.org/10.1016/j.exger.2005.10.008. Consider adding as a dbxref.

**Comment by dosumis** [comment]
This should be a superclass for "neuroblast (sensu arthropoda)"

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/cl-edit.owl:3534]: done!

**Comment by nicolevasilevsky** [comment]
> neuroblast (sensu arthropoda)

@dosumis, what is the ID for this class? I don't see it in CL.

**Review by bvarner-ebi — APPROVED** [review]

**Comment by nicolevasilevsky** [comment]
I'll merge this for now. @dosumis if there is further action needed, please let me know, thanks!

**Comment by cmungall** [comment]
> @dosumis, what is the ID for this class? I don't see it in CL.

I believe he meant this: CL:0000338 ! neuroblast (sensu Nematoda and Protostomia)

**Pull request merged**